src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/OneListPriceForAdditionalService.feature

```diff
@@ -0,0 +1,24 @@
+﻿Feature: One List Price For Additional Service Added
+    As a Buyer
+    I don't to want to have to select the list price of an Additional Service that has only one list price
+    So that I save time and effort when completing my order
+
+Background:
+	Given an incomplete order with catalogue items exists
+    When the Order Form for the existing order is presented
+    Then there is the Additional Service section
+	And the User is able to manage the Additional Services section
```

Comment:
The spacing doesn't seem to be consistent here

Reply:
sorting